deploy/devdeploy.py

```diff
@@ -42,7 +42,7 @@ def _copy_opscenter_files(cur, schema: str, stage: str):
     cur.execute(body)
 
 
-def devdeploy(profile: str, database: str, schema: str, stage: str):
```

Comment:
should schema be the same pattern as database?

Reply:
I didn't get this comment. Do you mean, just keep schema as some global variable with fix value and remove from function argument ?